packages/babel-parser/src/parser/comments.js

```diff
@@ -264,7 +264,17 @@ export default class CommentsParser extends BaseParser {
       ) {
         node.innerComments = trailingComments;
       } else {
-        node.trailingComments = trailingComments;
+        // TrailingComments maybe contain innerComments
+        const findIndex = trailingComments.findIndex(
+          comment => comment.end >= node.end,
+        );
+
+        if (findIndex !== -1 && findIndex !== 0) {
```

Comment:
Looks like we need tests for when:

* Trailing comments _only_ contain inner comments
* When will the second half of this (`findIndex !== 0`) evaluate truthy? Can we add a test for that?

Reply:
> Trailing comments only contain inner comments

Done.

> When will the second half of this (findIndex !== 0) evaluate truthy? Can we add a test for that?

Have it already, look like

```js
class A {
  test() {}
  // this.member = 'value';
}
```

if `findIndex === 0`mean  both **trailingComments**, so we can skip